src/main/webapp/app/quiz/edit/drag-and-drop-question/edit-drag-and-drop-question.component.ts

```diff
@@ -83,11 +92,18 @@ export class EditDragAndDropQuestionComponent implements OnInit, OnChanges, Afte
      */
     mouse: DragAndDropMouseEvent;
 
+    hintCommand = new HintCommand();
+    explanationCommand = new ExplanationCommand();
+
+    /** {array} with domainCommands that are needed for a drag and drop question **/
+    commandDragAndDropQuestions: DomainCommand[] = [this.explanationCommand, this.hintCommand];
```

Comment:
please refactor `commandDragAndDropQuestions` to `dragAndDropQuestionDomainCommands`